class ContactAlreadyExistsException(Exception):
    pass

class InvalidContactDataException(Exception):
    pass

class ContactNotFoundException(Exception):
    """Excepci√≥n lanzada cuando no se encuentra un contacto."""
    pass